Content.Server/Administration/Commands/BriefCommand.cs

```diff
@@ -0,0 +1,105 @@
+using System.Linq;
+using Content.Server.GameTicking;
+using Content.Server.Players;
+using Content.Shared.Administration;
+using Content.Shared.Roles;
+using Robust.Server.Player;
+using Robust.Shared.Console;
+using Robust.Shared.Prototypes;
+
+namespace Content.Server.Administration.Commands
+{
+    [AdminCommand(AdminFlags.Admin)]
+    public sealed class BriefCommand : IConsoleCommand
+    {
+        [Dependency] private readonly IEntityManager _entities = default!;
+        [Dependency] private readonly IPrototypeManager _prototypeManager = default!;
+
+        public string Command => "brief";
+        public string Description => "Makes you a mob of choice until the command is rerun.";
+        public string Help => $"Usage: {Command} <outfit> <name> <entity>";
+
+        public void Execute(IConsoleShell shell, string argStr, string[] args)
+        {
+            var player = shell.Player as IPlayerSession;
+            if (player == null)
+            {
+                shell.WriteLine("Nah");
+                return;
+            }
+
+            var mind = player.ContentData()?.Mind;
+
+            if (mind == null)
+            {
+                shell.WriteLine("You can't spawn here!");
+                return;
+            }
+
+            if (mind.VisitingEntity != default)
+            {
+                var entity = mind.VisitingEntity;
+                player.ContentData()!.Mind?.UnVisit();
+
+                foreach (var officer in _entities.EntityQuery<BriefOfficerComponent>(true))
+                {
+                    if (officer.Owner == entity) _entities.QueueDeleteEntity(officer.Owner);
+                }
+                return;
+            }
+
+            var coordinates = player.AttachedEntity != null
+                ? _entities.GetComponent<TransformComponent>(player.AttachedEntity.Value).Coordinates
+                : EntitySystem.Get<GameTicker>().GetObserverSpawnPoint();
+
+            string entname = "MobHuman";
+            if (args.Length >= 3) entname = args[2];
+
+            var brief = _entities.SpawnEntity(entname, coordinates);
```

Comment:
Validate that entname is valid instead of throwing when trying to spawn an invalid entity prototype.